packages/lesswrong/components/posts/BookmarkButton.tsx

```diff
@@ -13,10 +11,11 @@ import { useMutation, gql } from '@apollo/client';
 import * as _ from 'underscore';
 
 const styles = (theme: ThemeType): JssStyles => ({
-  icon: {
+  container: {
     cursor: "pointer",
     color: theme.palette.icon.dim3,
-  }
+  },
+  icon: theme.uiStyle === "friendly" ? {fontSize: "22px !important"} : {},
```

Comment:
Rather than using `!important`, I think we should set `stylePriority: -1` on `ForumIcon`